Consumer/ConsumerManager.php

```diff
@@ -0,0 +1,204 @@
+<?php
+namespace M6Web\Bundle\KafkaBundle\Consumer;
+
+use M6Web\Bundle\KafkaBundle\AbstractKafka;
+
+/**
+ * Class Consumer
+ * @package M6Web\Bundle\KafkaBundle\Consumer
+ *
+ * A class to consume messages with topics set or not
```

Comment:
Je suis pointilleux mais je verrais plutôt "A class to consume messages with set (or unset) topics"